python/GafferSceneUI/LightEditor.py

```diff
@@ -440,6 +440,37 @@ def __toggleBoolean( self, inspections ) :
 
 		return True
 
+	def __activeInspectionTweaks( self, pathListing ) :
+
+		selection = pathListing.getSelection()
+
+		columns = pathListing.getColumns()
+
+		tweaks = []
+
+		with Gaffer.Context( self.getContext() ) as context :
+			for i in range( 0, len( columns ) ) :
+				column = columns[ i ]
+				if not isinstance( column, _GafferSceneUI._LightEditorInspectorColumn ) :
+					continue
+				for path in selection[i].paths() :
+					context["scene:path"] = GafferScene.ScenePlug.stringToPath( path )
+					inspection = column.inspector().inspect()
+					if inspection is not None :
+						edit = inspection.acquireEdit()
+						if isinstance( edit, Gaffer.TweakPlug ) and edit["enabled"].getValue() :
+							tweaks.append( edit )
+
+		return tweaks
+
+	def __removeEdits( self, pathListing ) :
+
+		tweaks = self.__activeInspectionTweaks( pathListing )
```

Comment:
This needs to be in an UndoScope.

Reply:
Fixed in 0e0764df41d4f84d50da9438bde416e50b02b7c9